checkov/terraform/checks/graph_checks/gcp/GCPdisableAlphaClusterFeatureInKubernetesEngineClusters.yaml

```
@@ -0,0 +1,11 @@
+metadata:
+  id: "CKV2_GCP_19"
+  name: "Ensure GCP Kubernetes engine clusters have 'alpha cluster' feature disabled"
+  category: "KUBERNETES"
+
+definition:
+  cond_type: attribute
+  resource_types: "google_container_cluster"
+  attribute: "enable_kubernetes_alpha"
+  operator: "equals_ignore_case"
```

```suggestion
  operator: "not_equals_ignore_case"
```
it should not `true` right?

---

Oops, that's right. Making the changes now.